### PR TITLE
tutorial laravel: set max width to avoid large image display

### PR DESCRIPTION
**Pull request by askdkc**
Actual document screenshot size is just too big...
We need to add width to `` tag for better presentation.

image

### CODE REVIEW

**Comment by kou** [comment]
Can we use CSS instead of explicit `width=`?

**Comment by askdkc** [comment]
@kou san

Fixed. How about now?

**Comment by kou** [comment]
Ah, I meant https://github.com/pgroonga/pgroonga.github.io/blob/master/assets/stylesheets/pgroonga.scss not an inline CSS.

**Comment by askdkc** [comment]
@kou san

My bad. I add `img.laravel-image` to scss file and fixed the style.

Wish I could just use tailwindcss...

**Comment by kou** [comment]
How about this?

```scss
diff --git a/assets/stylesheets/pgroonga.scss b/assets/stylesheets/pgroonga.scss
index 8422b23..69de5d7 100644
--- a/assets/stylesheets/pgroonga.scss
+++ b/assets/stylesheets/pgroonga.scss
@@ -51,6 +51,10 @@ div.note {
   }
 }
 
+img {
+  @include img-responsive(block);
+}
+
 #myautocomplete {
   padding: 0 $padding-base-horizontal;
 }
```

**Comment by kou** [comment]
BTW, we should update our old Bootstrap (3.3.5) or replace other CSS framework. 

I think that Tailwind CSS isn't suitable for Jekyll. We don't want to write style in Markdown with inline HTML.
(I like Bulma if we want to change CSS framework from Bootstrap because Bulma doesn't require JavaScript.)

**Comment by askdkc** [comment]
@kou san

OK, I fixed it.

Tailwindcss is just my preference. Please ignore me on this one 😏

File: how-to/laravel.md [hunk 1]
```diff
@@ -583,20 +583,20 @@ php artisan serve
 
 Now you can access the application in your web browser at: [http://localhost:8000](http://localhost:8000)
 
-![Laravel sample](../images/laravel/laravel1.png)
+<img src="../images/laravel/laravel1.png" alt="Laravel sample">
```
Comment (kou) [inline how-to/laravel.md:586]: Could you revert inline HTMLs?

Comment (askdkc) [reply how-to/laravel.md]: Of course👍

And it's done!

**Comment by kou** [comment]
Thanks!

**Comment by askdkc** [comment]
Fix Release!! 🎉
Gundam.

https://www.reddit.com/r/Gundam/comments/13alt9j/oc_fix_release/

**Pull request merged**